portal/src/components/Header/Header.tsx

```diff
@@ -25,12 +24,15 @@ export const Header = ({ className }: { className?: string }) => {
         },
         className,
     );
+    const goToHome = () => {
+        window.location.href = "/";
+    };
 
     return (
         <header className={componentClassName}>
-            <Link onClick={() => setMenuIsOpen("")} to="/" className="jkl-portal-header__title">
+            <button onClick={() => goToHome()} className="jkl-portal-header__title">
```

Comment:
```suggestion
            <button onClick={goToHome} className="jkl-portal-header__title">
```

Dette gjør at du potensielt sparer noen rerenders og koden ser noen hakk bedre ut :)

Reply:
Men ville helst sett en `<Link to="/">`, så slippe vi å mutere location objektet direkte.

Reply:
Denne var allerede en `<Link to="/">` før denne endringen. Bakgrunnen for denne endringen er vel at når man allerede er på forsiden vil ikke `<Link>` gjøre noe, slik at det ikke skjer noe når man trykker på "Jøkul"-linken. Det er jo ikke noe problem i seg selv, men når man er på forsiden og har en fullskjermmeny åpen, kan det virke forvirrende at det ikke skjer noe.

Når det er sagt hadde jeg foretrukket en løsning der man fikk eventuelle åpne menyer til å lukke seg, heller enn å tvinge en navigasjon. `<button>` blir også feil semantisk element her, så hvis man finner en løsning som bruker `<a>` hadde det også vært fint.

Reply:
Testade flera alternativ för att få det å fungera "as expected" men Link + onClick hade en del issues som gjorde jag må finna en annan väg. Jag kan se på det igen.